pipelinit/pipelinit-cli
# Feat: Ruby add sast stage

Added a SAST (Static application security testing) stage for Ruby. In pipelinit, semgrep is the tool used for static analysis.

Resolves: #125 

To test:
In a GitHub repository with a ruby file, create a pull request of a branch. See the tests performed also in the Action tab.

We get this on GitHub PR:
![test2-sast by lilianeconstantino · Pull Request #2 · lilianeconstantino_spec - Google Chrome 29_12_2021 14_31_17 (2)](https://user-images.githubusercontent.com/81980069/147688646-119b00e5-0fde-4f17-8b93-2c36cc4c4d40.png)

And this on GitHub Actions:
![test-sast · lilianeconstantino_spec@b1f887e - Google Chrome 28_12_2021 15_58_42 (3)](https://user-images.githubusercontent.com/81980069/147606874-725ce114-410c-4414-b303-412d5605ff3a.png)
